app/src/main/java/com/lateinit/rightweight/data/database/dao/HistoryDao.kt

```diff
@@ -72,4 +79,13 @@ interface HistoryDao {
 
     @Query("DELETE FROM history")
     suspend fun removeAllHistories()
+
+    @Transaction
```

Comment:
insert 하나만 하면 transaction 할 필요 없을 거 같아요.
insert 어노테이션 자체에 transaction이 있다네요

> Putting this annotation on an [Insert](https://developer.android.com/reference/androidx/room/Insert), [Update](https://developer.android.com/reference/androidx/room/Update) or [Delete](https://developer.android.com/reference/androidx/room/Delete) method has no impact because those methods are always run inside a transaction.

https://developer.android.com/reference/androidx/room/Transaction

Reply:
이것도 제거하도록 하겠습니다!